thanks/index.md

```diff
@@ -0,0 +1,157 @@
+
+[GSoC]: https://summerofcode.withgoogle.com
+
+## Credits
+
+There are hundreds of people who have made wxWidgets possible through donated
+hardware, services, development, documentation, translations, and support. We
+couldn't possibly list them all here over the last twenty years, however, we
+would like to highlight just a few of those rockstars.
+
+### Developers
+
+We would particularly like to thank the following for their contributions to
+wxWidgets, and the many others who have been involved in the project over the
+years. Apologies for any unintentional omissions from this list.
+
+Main Authors:
+
+* wxWidgets 2 for GTK+: Robert Roebling - Latin scholar, medic, Renaissance man
+* wxWidgets 2 for Windows: Julian Smart - Multum in Parvo in Rutland
+* wxWidgets 2 for Motif: Julian Smart - didn't quite mean to do that port too,
+  oh well
+* wxWidgets 2 for Mac: Stefan Csomor - keeping baby/keyboard separation optimal
+* wxWidgets 1 for Xt: Markus Holzem - keeping his sanity now he's away
+* wxString, wxConfig, Windows drag and drop, and loads of other great stuff
+  without which we would be quite stuck: the one and only Vadim Zeitlin,
+  physicist and wxWidgets hero
+* wxThread, wxSocket and multimedia classes: the deceptively young but talented
+  Guilhem Lavaux
+* wxHTML: the highly dedicated Vaclav Slavik
+
+Other Contributors:
+
+Yiorgos Adamopoulos, Jamshid Afshar, Alejandro Aguilar-Sierra, AIAI, Patrick
+Albert, Karsten Ballueder, Michael Bedward, Kai Bendorf, Yura Bidus, Keith Gary
+Boyce, Chris Breeze, Pete Britton, Ian Brown, C. Buckley, Marco Cavallini,
+Dmitri Chubraev, Robin Corbet, Cecil Coupe, Andrew Davison, Neil Dudman, Robin
+Dunn, Hermann Dunkel, Jos van Eijndhoven, Tom Felici, Thomas Fettig, Matthew
+Flatt, Pasquale Foggia, Josep Fortiana, Todd Fries, Dominic Gallagher,
+Guillermo Rodriguez Garcia, Wolfram Gloger, Norbert Grotz, Stefan Gunter, Bill
+Hale, Patrick Halke, Stefan Hammes, Guillaume Helle, Harco de Hilster, Cord
+Hockemeyer, Olaf Klein, Leif Jensen, Bart Jourquin, Guilhem Lavaux, Jan
+Lessner, Nicholas Liebmann, Torsten Liermann, Per Lindqvist, Thomas Runge, Tatu
+Männistö, Scott Maxwell, Thomas Myers, Oliver Niedung, Stefan Neis, Hernan
+Otero, Ian Perrigo, Timothy Peters, Giordano Pezzoli, Harri Pasanen, Thomaso
+Paoletti, Garrett Potts, Marcel Rasche, Dino Scaringella, Jobst Schmalenbach,
+Arthur Seaton, Paul Shirley, Stein Somers, Petr Smilauer, Neil Smith, Kari
+Systä, Arthur Tetzlaff-Deas, Jonathan Tonberg, Jyrki Tuomi, Janos Vegh, Andrea
+Venturoli, David Webster, Xiaokun Zhu, Edward Zimmermann.
+
+### Other Support
+
+We would like to thank [AIAI][7], University of Edinburgh (and in particular
+Austin Tate and Robert Rae) for supporting the original work and giving
+permission to liberate the wxWidgets source.
+
+[7]: http://www.aiai.ed.ac.uk/
```

Comment:
Perhaps this URL would be better
https://web.inf.ed.ac.uk/aiai

Reply:
yes, thanks, I'll use that